sdk/identity/identity/src/credentials/interactiveBrowserCredential.ts

```diff
@@ -116,71 +120,101 @@ export class InteractiveBrowserCredential implements TokenCredential {
     // eslint-disable-next-line
     return new Promise<AccessToken | null>(async (resolve, reject) => {
```

Comment:
Because this method just returns the promise constructor, `acquireTokenFromBrowser` doesn't need to be async, it can just return a Promise. If you remove that, I bet you can get rid of the eslint disable too!